src/lib/realmdb/RealmDB.js

```diff
@@ -86,7 +86,7 @@ class RealmDB implements DB, ProfileDB {
 
       // `App.currentUser` updates to match the logged in user
       log.debug('realm logged in', { user: this.user })
-      this._syncFromRemote()
+      await this._syncFromRemote()
```

Comment:
```suggestion
     this._syncFromRemote()
```

Reply:
@sirpy why do we skipping await ? you want to do it 'in background' ?
in such case need to make sure we won't have unhandled promise rejection
all errors should be caunght inside _syncFromRemote or .catch() block should be added

Reply:
yes. ok please handle exceptions